code/src/Components/ThirdQuestion.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+const ThirdQuestion = ({ homeInput, onHomeInputChange, onStepChange }) => {
+  return (
+    <section className='questionWrapper'>
+      <div className='question'>
+        <form>
+          <h2 tabindex='0'>In which dormitory would you stay?</h2>
+          <div className='radioGroup'>
+            <label className='dormitory'>
+              <input
+                type='radio'
+                name='dormitorys'
+                value='Slytherin'
+                onChange={onHomeInputChange}
+                defaultChecked={homeInput === 'Slytherin'}
+              />
+              Slytherin
+            </label>
```

Comment:
A suggestion for projects in the future is to create an array with the different options and then map through them, in that way you don't have to repeat yourself so much :D Something like
`const homeArray = ['Slytherin', 'Ravenclaw', 'Gryffindor', 'Hufflepuff']` and then use `{homeArray.map(() => (
))}` to go through the array.

Reply:
Thanks for the tip! Didn't think about that :D